autorel/clients/base.py

```diff
@@ -37,6 +40,44 @@ def __init__(self,endpoint,port,timeout=None):
         self._port = port
         self._timeout = timeout if timeout else DEFAULT_TIMEOUT
         self._headers = DEFAULT_HEADERS
+        self._startDebugging()
+
+    def _startDebugging(self):
```

Comment:
I wouldn't implement a debugger logic (frameCount, frameBuffer, ...) here, it should be a separate class with more general methods.